.github/workflows/build_wheels.yml

```diff
@@ -13,28 +13,17 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        os: [ubuntu-20.04, macOS-10.15]
 
     steps:
       - uses: actions/checkout@v2
 
       - name: Install Python
         uses: actions/setup-python@v2
-        with:
-          python-version: '3.7'
 
-      - name: Install cibuildwheel
-        run: |
-          python -m pip install cibuildwheel
-      - name: Build wheels for CPython
-        run: |
-          python -m cibuildwheel --output-dir dist
-        env:
-          CIBW_BUILD: "cp37-*64 cp38-*64 cp39-*64"
-          CIBW_MANYLINUX_X86_64_IMAGE: manylinux1
-          CIBW_BEFORE_BUILD: pip install .
+      - name: Build wheel
```

Comment:
the recipe on the left breaks, not sure how it can work in fairseq, but in our case we would need machines with cuda anyway. The current version should work but it will not have the cuda files (the wheels are not used, it should be part of a follow up work so that we automate the releases, not really ready right now)